script/constructron.lua

```diff
@@ -446,9 +385,7 @@ me.do_until_leave = function(job)
                     end
                 end
                 local next_station = me.get_closest_unused_service_station(job.constructrons[1], job.unused_stations)
-                for _, constructron in ipairs(job.constructrons) do
-                    me.pathfinder.request_path({constructron}, "constructron_pathing_dummy" , next_station.position)
-                end
+                pathfinder.init_path_request(job.constructrons[1], next_station.position, job)

```

Comment:
could also replace `job.constructrons[1]` here like in the other places (also above here, couldn't add those lines to this comment)